tests/test_rooms/test_create_room.py

```diff
@@ -101,3 +102,13 @@ def test_room_creation_with_invalid_calendar_id(self):
             room_invalid_calendar_id_mutation_query,
             room_invalid_calendar_id_mutation_response
         )
+
+    def test_room_creation_with_non_existing_tag(self):
+        """
+        Test room creation with  invalid tag
+        """
+        CommonTestCases.admin_token_assert_in(
+            self,
+            non_existing_tag_mutation,
```

Comment:
Since we know all the tags that are supposed to be there by name, use tag names in your error messages and avoid using integers to add more clarity to users.

Reply:
@Bonifase, your concerns are noted. However, this might nit be applicable when a tag does not exist completely, the tag does not have a name, only way to raise an error is to use the integer the user had input.

Reply:
This has been updated to "Tag id" for more clarity @Bonifase, since it is impossible to use tag name for non-existent tags.